packages/terra-data-grid/src/subcomponents/Cell.module.scss

```diff
@@ -4,15 +4,15 @@
 @import '../clinical-lowlight-theme/Cell.module';
 
 :local {
-  .worklist-data-grid-cell {
-    border-bottom: var(--terra-worklist-data-grid-cell-border-bottom, 1px solid #dedfe0);
-    border-right: var(--terra-worklist-data-grid-cell-border-right, 1px solid #dedfe0);
+  .data-grid-cell {
+    border-bottom: var(--terra-data-grid-cell-border-bottom, 1px solid #dedfe0);
+    border-right: var(--terra-data-grid-cell-border-right, 1px solid #dedfe0);
     font-weight: normal;
-    padding: var(--terra-worklist-data-grid-cell-content-padding, 7px);
+    padding: var(--terra-data-grid-cell-content-padding, 7px);
     position: relative;
 
     &:first-child {
-      border-left: var(--terra-worklist-data-grid-cell-border-left, 1px solid #dedfe0);
+      border-left: var(--terra-data-grid-cell-border-left, 1px solid #dedfe0);
     }
 
     input {
```

Comment:
The Cell component should not be controlling the styles for the input element.  This is a RowSelectionCell concern.  It should not be part of the Cell component sass.